rshell/main.py

```diff
@@ -932,6 +932,7 @@ def rsync(src_dir, dst_dir, mirror, dry_run, print_func, recursed, sync_hidden):
                       "'{}' is a directory. Ignoring"
                 print_err(msg.format(src_filename, dst_filename))
             else:
+                print_func('Checking {}'.format(dst_filename))
```

Comment:
This line doesn't seem to be related to the PR

Reply:
That's true. I found it convenient in the context of this change request to give the user some feedback what is going on. We are syncing the pretty large codebase of the [Terkin-Datalogger](https://github.com/hiveeyes/terkin-datalogger) which will take some time to upload. Even if no files have been changed, it will take some time.

Please let me know if you accept this or whether you want me to remove this line.

Reply:
It's not relevant to this particular PR, so it should be removed. It's probably worth adding, but should be done in a separate PR.

Reply:
All right. I've removed the line as requested. Thanks again!